noir-projects/aztec-nr/aztec/src/encrypted_logs/log_type.nr

```diff
@@ -0,0 +1,3 @@
+pub global PRIVATE_NOTE_LOG_TYPE_ID: u64 = 0;
+pub global PARTIAL_NOTE_LOG_TYPE_ID: u64 = 1;
```

Comment:
```suggestion
pub global PARTIAL_NOTE_PRIVATE_LOG_TYPE_ID: u64 = 1;
```

Partial note shave both private and public logs, this is the id for the private log.